backend/de.metas.contracts/src/main/sql/postgresql/system/50-de.metas.contracts/5628015_sys_gh12598_CallOrder_Unq_Index.sql

```diff
@@ -0,0 +1,11 @@
+CREATE UNIQUE INDEX unq_callorder_orderline ON c_callorderdetail (C_OrderLine_ID)
+;
+
+CREATE UNIQUE INDEX unq_callorder_inoutline ON c_callorderdetail (M_InOutLine_ID)
```

Comment:
i think this won't work, since one order line might have `n` shpment or invoice lines